charts/nginx-gateway-fabric/Chart.yaml

```diff
@@ -4,6 +4,7 @@ description: NGINX Gateway Fabric
 type: application
 version: 1.2.0
 appVersion: "edge"
+kubeVersion: ">= 1.25.0-0"
```

Comment:
How does this affect previous versions of NGF? Does this only apply for edge?

Reply:
yeah until we release it's only for `edge`